`@opentelemetry/instrumentation-graphql` Version `0.27.1` Missing `@types/graphql` dependency

### What version of OpenTelemetry are you using?
1.0.4

### What version of Node are you using?
v16.11.0

### Please provide the code you used to setup the OpenTelemetry SDK
```ts
import { GraphQLInstrumentation } from "@opentelemetry/instrumentation-graphql";
```

### What did you do?
In a fresh folder
```bash
yarn add @opentelemetry/api 
yarn add @opentelemetry/instrumentation-graphql
yarn add -D typescript
```
Add the build script to your `package.json`
```js
{
  "scripts": {
    "build": "tsc otel.ts"
  },
  "devDependencies": {
    "typescript": "^4.5.4"
  },
  "dependencies": {
    "@opentelemetry/api": "^1.0.4",
    "@opentelemetry/instrumentation-graphql": "^0.27.1"
  }
}
```
Attempt to compile the typescript
```bash
yarn build
```

### What did you expect to see?
I expected the file to compile successfully

### What did you see instead?
The file failed to compile because the graphql dependency was not installed
```
$ tsc otel.ts
node_modules/@opentelemetry/instrumentation-graphql/build/src/instrumentation.d.ts:2:36 - error TS2307: Cannot find module 'graphql' or its corresponding type declarations.

2 import type * as graphqlTypes from 'graphql';
```

### Additional context
In Version `0.26.1` it contains the dependency `"@types/graphql": "14.5.0"` which has somehow been dropped in the latest version

--- a/plugins/node/opentelemetry-instrumentation-graphql/package.json
+++ b/plugins/node/opentelemetry-instrumentation-graphql/package.json
@@ -53,7 +53,6 @@
     "@types/mocha": "8.2.3",
     "@types/node": "14.17.9",
     "codecov": "3.8.3",
-    "graphql": "15.5.1",
     "gts": "3.1.0",
     "mocha": "7.2.0",
     "nyc": "15.1.0",
@@ -63,6 +62,7 @@
     "typescript": "4.3.5"
   },
   "dependencies": {
-    "@opentelemetry/instrumentation": "^0.27.0"
+    "@opentelemetry/instrumentation": "^0.27.0",
+    "graphql": "^15.5.1"
   }
 }

